### DISCUSSION TITLE
Payment trouble

### DISCUSSION THREAD

**Discussion by GregKuklin** [TownyAdvanced/Towny#6958] [Q&A] [answered]
Привет! Hello! I wanna buy sponsorship on github to receive access on EventWars and other plugins, but I have little problem. My country(Russian Federation) is under sanctions on the github, so I can buy sponsorship on github. I'm asking you to help me. Are there any other payment methods? I can pay double price, just help me please!

**Comment by LlmDl** [comment, marked as answer]
You could find a non-Russian to sponsor for you, or you could move out of Russia.
